qutebrowser/browser/webengine/spell.py

```diff
@@ -0,0 +1,150 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2017 Michal Siedlaczek <michal.siedlaczek@gmail.com>
+
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Installing and configuring spell-checking for QtWebEngine."""
+
+import os
+from urllib.parse import urljoin
+from urllib.request import urlretrieve
+
+from PyQt5.QtCore import QLibraryInfo
+
+repository_url = 'https://redirector.gvt1.com/edgedl/chrome/dict/'
```

Comment:
Instead, I think it should get the file list from [this google API link](https://chromium.googlesource.com/chromium/deps/hunspell_dictionaries.git/+/master/?format=JSON) (giving you [this list](https://chromium.googlesource.com/chromium/deps/hunspell_dictionaries.git/+/master/) as JSON), and then download the files from there (rather than using an internal google host which might change).